Library/ViewModels/BackerDashboardProjectsViewModelTests.swift

```diff
@@ -55,8 +55,8 @@ internal final class BackerDashboardProjectsViewModelTests: TestCase {
       self.emptyStateProjectsType.assertValues([.backed])
       self.isRefreshing.assertValues([true, false])
 
-      self.vm.inputs.viewWillAppear(false)
-      self.isRefreshing.assertValues([true, false, true])
+      self.vm.inputs.viewWillAppear(true)
+      self.isRefreshing.assertValues([true, false, true], "Projects refresh after swipe animation.")
```

Comment:
It does keep your place after paginating and swiping between tabs (i've also tested swiping with paginating through the project navigator and all seems well), but I will doublecheck paginating several times with swipes.